testdata/cheats/Json.t.sol

```diff
@@ -240,6 +240,17 @@ contract WriteJsonTest is DSTest {
         vm.removeFile(path);
     }
 
+    function test_serializeRootObject() public {
```

Comment:
let's maybe add a comment to this test linking to the relevant issue and how this functionality was added in this PR

Reply:
done!